src/DynamoPackages/PackageManagerSearchElement.cs

```diff
@@ -135,6 +189,41 @@ internal set
             }
         }
 
+        private List<VersionInformation> versionDetails;
+        /// <summary>
+        /// A property for package Versions details (+ compatibility info)
+        /// </summary>
+        public List<VersionInformation> VersionDetails
+        {
+            get { return versionDetails; }
+            set
+            {
+                versionDetails = value;
+                LatestCompatibleVersion = GetLatestCompatibleVersion();
+                RaisePropertyChanged(nameof(VersionDetails));
+            }
+        }
+
+        /// <summary>
+        /// The currently selected version - need to pass onto the PackageDetailView
+        /// </summary>
+        public VersionInformation SelectedVersion { get; internal set; }
+
+        // Compatibility mapping for host and versions (to be replaced - fetch from the '/host_map' route)
+        private static Dictionary<string, Dictionary<string, string>> compatibilityMap = new Dictionary<string, Dictionary<string, string>> {
+            { "Revit", new Dictionary<string, string> {
+                {"2016", "1.3.2"}, {"2017", "2.0.2"}, {"2018", "2.0.2"}, {"2019", "2.0.2"},
+                {"2020", "2.1.0"}, {"2020.1", "2.2.1"}, {"2020.2", "2.3.0"}, {"2021", "2.5.2"},
+                {"2021.1", "2.6.1"}, {"2022", "2.10.1"}, {"2022.1", "2.12.0"}, {"2023", "2.13.1"},
+                {"2023.1", "2.16.1"}, {"2023.1.3", "2.16.2"}, {"2024", "2.17.0"}, {"2024.1", "2.18.1"},
+                {"2024.2", "2.19.3"}, {"2025", "3.0.3"}, {"2025.1", "3.0.3"}, {"2025.2", "3.2.1"}
+            }},
+            { "Civil3D", new Dictionary<string, string> {
+                {"2020", "2.1.1"}, {"2020.1", "2.2.0"}, {"2020.2", "2.4.1"}, {"2021", "2.5.2"},
+                {"2022", "2.10.1"}, {"2023", "2.13.1"}, {"2024", "2.17.1"}, {"2024.1", "2.18.1"},
+                {"2024.2", "2.18.1"}, {"2024.3", "2.19"}, {"2025", "3.0.3"}, {"2025.1", "3.2.2"}
+            }}
+        };
```

Comment:
We can use the latest Greg nuget with the new API now available for host mappings.https://www.nuget.org/packages/Greg/3.0.2.6533